services/frontend/src/request.ts

```diff
@@ -1,11 +1,15 @@
 import {
   ApiJob,
   ApiLoginResponse,
+  ApiModel,
   ApiSignupResponse,
+  ApiSolver,
   AuthServiceForm,
 } from "./types";
 
-const apiServiceUrl = "/api/v1";
+const apiServiceUrl = import.meta.env.DEV
+  ? "http://project.127.0.0.1.sslip.io/api/v1"
```

Comment:
Dette vil ikke virke på Linux. Hvorfor ikke bare altid bruge `/api/v1`, det bør vel også virke i development?

Reply:
Godt spottet, denne ændring overså jeg fuldstændigt.

Reply:
det er når man kører en local dev server

jeg ligger en kommentar i readme

Reply:
Ah ja, det giver mening. Er det måske så muligt at ændre strengen til `http://${import.meta.env.PROJECT_DOMAIN}/api/v1` for local development, så vi ikke låser det fast?

Reply:
jeg gør det configurable og skriver det i README'en.